pom.xml

```diff
@@ -15,28 +15,46 @@
     <description>CHS API handling CRUD operations for ordering missing images through the Companies House Direct (CHD) database</description>
 
     <properties>
-        <commons.lang.version>2.6</commons.lang.version>
-        <java.version>1.8</java.version>
-        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
-        <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-        <org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
-        <oracle.jdbc8.version>12.2.0.1.0</oracle.jdbc8.version>
-        <spring-boot-dependencies.version>2.5.5</spring-boot-dependencies.version>
-        <spring-boot-maven-plugin.version>2.5.5</spring-boot-maven-plugin.version>
+        <java.version>21</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven-compiler-plugin.version>3.12.1</maven-compiler-plugin.version>
+        <maven-surefire-plugin.version>3.2.5</maven-surefire-plugin.version>
+
+        <org.mapstruct.version>1.5.5.Final</org.mapstruct.version>
+        <oracle.jdbc11.version>23.4.0.24.05</oracle.jdbc11.version>
+
+        <spring-boot-dependencies.version>3.2.4</spring-boot-dependencies.version>
+        <spring-boot-maven-plugin.version>3.2.4</spring-boot-maven-plugin.version>
+        <spring-web-version>6.1.6</spring-web-version>
         <start-class>uk.gov.companieshouse.chd.order.api.ChdOrderApiApplication</start-class>
-        <structured-logging.version>1.9.12</structured-logging.version>
-        <environment-reader-library.version>1.3.6</environment-reader-library.version>
-        <system-rules-version>1.17.2</system-rules-version>
+
+        <environment-reader-library.version>3.0.1</environment-reader-library.version>
+
+        <system-rules-version>1.19.0</system-rules-version>
+
+
         <hamcrest-all-version>1.3</hamcrest-all-version>
```

Comment:
Can we upgrade this to new hamcrest dependency and use version 2.2, hamcrest-all is depreciated.

https://mvnrepository.com/artifact/org.hamcrest/hamcrest/2.2

Reply:
Actually in virtual repository latest version still 1.3 
http://repository.aws.chdev.org:8081/artifactory/virtual-release/org/hamcrest/hamcrest-all/

Reply:
Done